server/etc/e-smith/templates/var/lib/nethserver/sieve-scripts/before.sieve/10junkmail

```diff
@@ -3,21 +3,32 @@
 # 10junkmail 
 #
 {
-     if($dovecot{SpamFolder} ne '') {
-        $OUT = <<EOF
+my $subject = $rspamd{SpamSubjectPrefixString} || '***SPAM***';
+my $spamfolder = $dovecot{SpamFolder} || '';
+
+if ( $spamfolder ne '') {
+
+    $OUT .= qq(
 # -- enabled (move marked SPAM messages into junkmail folder)
 require ["fileinto", "mailbox"];
 if header :contains "X-Spam-Flag" "YES" {
     fileinto :create "$dovecot{SpamFolder}";
     stop;
 }
-elsif header :contains "subject" "$rspamd{SpamSubjectPrefixString}" {
-    fileinto :create "$dovecot{SpamFolder}";
-    stop;
+);
+
+    if ( -f "/etc/e-smith/db/configuration/defaults/rspamd/type") {
```

Comment:
If the condition is not satisfied the whole fragment should return an empty string